lucille-core/src/main/java/com/kmwllc/lucille/util/SolrUtils.java

```diff
@@ -53,6 +52,33 @@ public static SolrClient getSolrClient(Config config) {
     }
   }
 
+  private static CloudSolrClient getCloudClient(Config config) {
+    // todo review if zkHosts is required
```

Comment:
I think we should reflect what SolrJ supports.
Since it allows for creating a CloudSolrClient without zkHosts, it's OK for us to allow the same
It would be useful to have some INFO level logging about what kind of connection is being created and with what parameters -- but that could be left as a separate task.
Tagging @kyle-kmw to get his opinion too

Reply:
I almost added a comment asking to force it to be required but I ultimately didn't with the same reasoning as @rseitz . Adding logging as a separate task (separate PR) makes sense to me. I am thinking that for the case where the indexer is configured with solr urls in cloud mode we should add a WARN level log entry that using zk based connections will be more reliable.

Reply:
removed the todo comment and added [LC-336](https://kmwllc.atlassian.net/browse/LC-336) to backlog